vision/spring-framework/pom.xml

```diff
@@ -0,0 +1,90 @@
+<!--
+Copyright 2019 Google Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<project>
+	<modelVersion>4.0.0</modelVersion>
+
+	<properties>
+		<maven.compiler.target>1.8</maven.compiler.target>
+		<maven.compiler.source>1.8</maven.compiler.source>
+	</properties>
+
+	<!--
+		The parent pom defines common style checks and testing strategies for our samples.
+		Removing or replacing it should not affect the execution of the samples in anyway.
+	-->
+	<parent>
+		<groupId>com.google.cloud.samples</groupId>
+		<artifactId>shared-configuration</artifactId>
+		<version>1.0.11</version>
+	</parent>
+
+	<groupId>com.example.vision</groupId>
+	<artifactId>spring-framework</artifactId>
+	<name>Spring Framework with Cloud Vision Sample</name>
+
+	<!-- [START spring_vision_bom_fragment] -->
+	<dependencyManagement>
+		<dependencies>
+			<dependency>
+				<groupId>org.springframework.cloud</groupId>
+				<artifactId>spring-cloud-gcp-dependencies</artifactId>
+				<version>1.1.2.RELEASE</version>
+				<type>pom</type>
+				<scope>import</scope>
+			</dependency>
+		</dependencies>
+	</dependencyManagement>
+	<!-- [END spring_vision_bom_fragment] -->
+
+	<dependencies>
+		<!-- [START spring_vision_dependency_fragment] -->
+		<dependency>
+			<groupId>org.springframework.cloud</groupId>
+			<artifactId>spring-cloud-gcp-starter-vision</artifactId>
+		</dependency>
+
+		<dependency>
+			<groupId>org.springframework.cloud</groupId>
+			<artifactId>spring-cloud-gcp-starter-storage</artifactId>
+		</dependency>
+		<!-- [END spring_vision_dependency_fragment] -->
+
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter-web</artifactId>
+			<version>2.1.6.RELEASE</version>
```

Comment:
Can we import some spring boot bom to get  these versions automatically?

Reply:
I could; however I actually prefer this way because it makes the spring-cloud-gcp-dependencies the only dep in the BOM for this sample, and this will make the BOM code snippet more clear (since the user following along will only see `spring-cloud-gcp-dependencies` and not worry about what is `spring-boot-dependencies`).

Reply:
As it is today, you can actually just include the Spring Cloud BOM which contains `spring-cloud-gcp-dependencies`, but that's probably not what you want in the code snippet.
If nothing else, I would at least define the Spring Boot version `2.1.6.RELEASE` as a property at the top, so that you don't repeat it multiple times in the file.

Reply:
Sounds good; I ended up adding the property for the version.